hbase-client/src/main/java/org/apache/hadoop/hbase/ipc/AbstractRpcClient.java

```diff
@@ -188,14 +186,6 @@ public AbstractRpcClient(Configuration conf, String clusterId, SocketAddress loc
 
     this.connections = new PoolMap<>(getPoolType(conf), getPoolSize(conf));
 
-    this.cleanupIdleConnectionTask = IDLE_CONN_SWEEPER.scheduleAtFixedRate(new Runnable() {
```

Comment:
We have another rpc client implementation which depends on this I believe?

Reply:
I didn't find any at the time i was doing the patch

Reply:
It's BlockingRpcClient.

Reply:
Just committed a version that restore that. To avoid having it done twice (in the case of Netty), i add hasIdleCleanupSupport() where needed